scio-parquet/src/main/scala/com/spotify/scio/parquet/avro/syntax/SCollectionSyntax.scala

```diff
@@ -73,29 +48,27 @@ class SCollectionOps[T](private val self: SCollection[T]) extends AnyVal {
    * @param compression
    *   defaults to snappy
    */
-  def saveAsDynamicParquetAvroFile(
+  def saveAsParquetAvroFile(
```

Comment:
This is a breaking change, the old `saveAsDynamicParquetAvroFile` was actually _not_ a 'dynamic' per the usage of that term in the rest of the API. Also the API was a little _too_ scala-y; requiring an `Either`.

Reply:
I think it's ok for this to be a breaking change; it doesn't seem to have much, if any, adoption in its current state :)